cfme/fixtures/appliance.py

```diff
@@ -184,3 +186,63 @@ def temp_appliances_unconfig_funcscope(appliance, pytestconfig):
             preconfigured=False
     ) as appliances:
         yield appliances
+
+
+@pytest.fixture(scope="function")
+def vddk_url(provider):
+    try:
+        major, minor = str(provider.version).split(".")
+    except ValueError:
+        major = str(provider.version)
+        minor = "0"
+    vddk_version = "v{}_{}".format(major, minor)
+    # cf. BZ 1651702 vddk_version 6_7 does not currently work with CFME, so use v6_5
+    if BZ(1651702, forced_streams=['5.9', '5.10']).blocks:
+        vddk_version = "v6_5"
+    try:
+        url = conf.cfme_data.basic_info.vddk_url.get(vddk_version)
+    except (KeyError, AttributeError):
+        pytest.skip('VDDK URL/Version not found in cfme_data.basic_info')
+    if url is None:
+        pytest.skip("There is no vddk url for this VMware provider version")
+    else:
+        return url
+
+
+@pytest.fixture(scope="module")
+def vddk_url_modscope(provider):
+    try:
```

Comment:
The code of these two methods looks identical to me? Am I missing a slight difference?

If so, let's write one method, and call it from both fixture methods.